### PR TITLE
Application for MigrationEase.md

### PR DESCRIPTION
**Pull request by ainhoa-a**
#### Project Abstract

The primary goal of this project is to create an intuitive, user-friendly interface that simplifies the migration process for Polkadot and Parachain Ledger users. 

This web application will allow connecting  Ledger devices, automatically display accounts eligible for migration, and assist users through the migration process in a secure and clear manner. 

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [x] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (Polkadot AssetHub (USDC & DOT) address in the application and bank details via email, if applicable).
- [x] I understand that an agreed upon percentage of each milestone will be paid in vested DOT, to the Polkadot address listed in the application.
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [x] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@ainhoa_a:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Thanks for the application @ainhoa-a few quick points:

- We haven't offered ETH mainnet payments for some time now. Could you please provide an AssetHub USDC address instead?
- We now require grantees to accept at least 50% of payment in vested DOT (linearly on-chain for two years). Would this be an issue for you? If not can you please provide a DOT address as well?
- Milestone 3 isn't clear to me, does this mean that there would essentially be a maintenance period of 12 months, with a delivery each month? If I understand correctly, you are asking for a monthly payment of $1k to equal a sum of $12k after 12 months? Could you also add some deliverables to this milestone?

**Comment by ainhoa-a** [comment]
Hi @keeganquigley, 

Here the DOT/AssetHub address **1fN87Fgj5BUhezFgbLiGbXTMrBVggnmYBX9anzMBky8KaJ5**

> * We now require grantees to accept at least 50% of payment in vested DOT (linearly on-chain for two years). Would this be an issue for you? If not can you please provide a DOT address as well? 

Yes, this won't be an issue. 

> * Milestone 3 isn't clear to me, does this mean that there would essentially be a maintenance period of 12 months, with a delivery each month? If I understand correctly, you are asking for a monthly payment of $1k to equal a sum of $12k after 12 months? Could you also add some deliverables to this milestone?

Yes, a monthly payment of $1k per month. We cover all costs for the operation of the website, we can provide monitoring information similar to checkly.

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Thanks @ainhoa-a for the response, but could you please update the address in the application itself? Thanks!

**Comment by ainhoa-a** [comment]
Application has been updated

**Comment by keeganquigley** [comment]
Thanks @ainhoa-a I will mark as ready for review and ping the rest of the committee for comment.

**Review by keeganquigley — CHANGES_REQUESTED** [review]
I just have one question for now, which is, how many users do you anticipate will need to utilize this migration app? Could you provide some evidence such as user feedback which shows that there is currently a huge need for it?

My concern is that it is a lot of funding for a one-time use app that maybe only a hundred people will use. Thanks for any further insights.

**Review by semuelle — CHANGES_REQUESTED** [review]
Hi @ainhoa-a, thanks for the application. I think this project makes sense. In case you haven't noticed, public grants are not paid partially in vested DOT (see [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md?plain=1#L127)), though. Could you add this line to the application?

**Comment by semuelle** [comment]
pinging @ainhoa-a

**Comment by ainhoa-a** [comment]
> Hi @ainhoa-a, thanks for the application. I think this project makes sense. In case you haven't noticed, public grants are not paid partially in vested DOT (see [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md?plain=1#L127)), though. Could you add this line to the application?

Thanks @semuelle for your feedback, and sorry for the late reply I'm currently on vacation. Yes we're ok with the vested DOTs, I updated the application.

**Comment by ainhoa-a** [comment]
> I just have one question for now, which is, how many users do you anticipate will need to utilize this migration app? Could you provide some evidence such as user feedback which shows that there is currently a huge need for it?
> 
> My concern is that it is a lot of funding for a one-time use app that maybe only a hundred people will use. Thanks for any further insights.

Hi @keeganquigley I understand your concerns, and we are the first that would like to see the application be used by many.  We do not have today a concrete number of users as Ledger does not share usage numbers, but every Ledger user of Kusama, System Chains or any parachain that previously had a Ledger app (>10) will need to migrate and today this process is very unfriendly and time consuming.

Eventually we believe this app could be extended to other usecases. For instance, mnemonic migrations, a central point for "recovering" funds from parachains that go unsupported or for users that are looking to consolidate all their parachain assets into DOTs to name a few ideas.

**Comment by keeganquigley** [comment]
Thanks for your answers @ainhoa-a makes sense. I heard that an additional feature may have been recently requested by @michalisFr, are you planning to make any changes to the current application? Additionally, would you be willing to receive milestone 3 payment all at once? Or would you rather stick with monthly payments of $1k?

**Comment by semuelle** [comment]
@ainhoa-a, feel free to continue the conversation in the chat room linked above. We didn't notice the request for private conversation earlier, sorry about that.

File: applications/MigrationEase.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+| 2c. | Code| Simplified Management of proxies, multisig and decision deposits  |
+| 2d. | Code| Standardized configuration and documentation |
+| 2e. | Code| Enhanced User Experience |
+
+### Milestone3  Maintenance and improvements 
+
+- **Estimated workload:** 1 month 
+- **Delivery time:**: 12 months 
+- **FTE:**  0,3 
+- **Costs:** 12'000/ USDT - We request a monthly payment of $1k per month.
```
Comment (semuelle) [inline applications/MigrationEase.md:166]:
```suggestion
- **Costs:** 12'000/ USDT - We request a payment of $3k per quarter.
```

**Review by semuelle — APPROVED** [review]

**Review by keeganquigley — APPROVED** [review]
Thanks for the additions @ainhoa-a LGTM

**Review by Noc2 — APPROVED** [review]

**Review by PieWol — APPROVED** [review]

**Review by SBalaguer — APPROVED** [review]

**Comment by keeganquigley** [comment]
Hi @ainhoa-a I believe M1 is about 5 months overdue, can you provide a progress update please?

**Comment by ainhoa-a** [comment]
Hi @keeganquigley M1 has been delivered here https://github.com/w3f/Grant-Milestone-Delivery/pull/1259

**Comment by ainhoa-a** [comment]
@PieWol  @keeganquigley  we submitted M2 last week https://github.com/w3f/Grant-Milestone-Delivery/pull/1272 looking forward to your feedback

**Pull request merged**